app/views/users/new.html.erb

```diff
@@ -1,44 +1,37 @@
 <div class="container">
+  <br><br><br><br><br>
+  <h1 class="col-xs-12 col-sm-offset-2 col-sm-8">Sign in please</h1>
 
-  <h3>Sign in please</h3>
-
-  <div class="well">
+  <div class="col-xs-12 col-sm-offset-2 col-sm-8 sign_in_form">
+    <br>
     <%= form_for(setup_user(@user)) do |f|  %>
       <div class="form-group">
-        <%= f.label :first_name %>
-        <%= f.text_field :first_name, class: 'form-control' %>
+        <%= f.text_field :first_name, placeholder: 'First name', class: 'form-control' %>
       </div>
       <div class="form-group">
-        <%= f.label :last_name %>
-        <%= f.text_field :last_name, class: 'form-control' %>
+        <%= f.text_field :last_name, placeholder: 'Last name', class: 'form-control' %>
       </div>
       <div class="form-group">
-        <%= f.label :email %>
-        <%= f.email_field :email, class: 'form-control' %>
+        <%= f.email_field :email, placeholder: 'Email', class: 'form-control' %>
       </div>
       <div class="form-group">
-        <%= f.label :password %>
-        <%= f.password_field :password, class: 'form-control' %>
+        <%= f.password_field :password, placeholder: 'Password', class: 'form-control' %>
       </div>
       <div class="form-group">
-        <%= f.label :password_confirmation %>
-        <%= f.password_field :password_confirmation, class: 'form-control' %>
+        <%= f.password_field :password_confirmation, placeholder: 'Password confirmation', class: 'form-control' %>
       </div>
       <%= f.fields_for :address do |ff| %>
         <div class="form-group">
-          <%= ff.label :line_1 %>
-          <%= ff.text_field :line_1, class: 'form-control' %>
+          <%= ff.text_field :line_1, placeholder: 'Line 1', class: 'form-control' %>
         </div>
         <div class="form-group">
-          <%= ff.label :postal_code %>
-          <%= ff.text_field :postal_code, class: 'form-control' %>
+          <%= ff.text_field :postal_code, placeholder: 'Postal code', class: 'form-control' %>
         </div>
         <div class="form-group">
-          <%= ff.label :city %>
-          <%= ff.text_field :city, class: 'form-control' %>
+          <%= ff.text_field :city, placeholder: 'City', class: 'form-control' %>
         </div>
         <div class="form-group">
-          <%= ff.label :province %>
+          <!-- <%= ff.label :province %> -->
           <%= ff.select(:province, options_for_select([
```

Comment:
weird that this works, I was under the assumption erb code inside html comments would be still be run, but I guess in this case it only generates an html label, which then is commented out. So yeah, does make sense actually.